spec/README.mediawiki

```diff
@@ -1190,23 +1277,31 @@ for verification and relay to the counterparty.
 |-
 | timestamp  || int    || UTC timestamp. order broadcast time
 |-
-| sig        || string || client signature of the serialized progress notification. serialization described below
+| sig        || string || client signature of the serialized notification. serialization described below
 |}
 
-'''DEX Acknowledgement'''
+'''Redeem notification serialization'''
 
 {|
-! field      !! type   !! description
+! field      !! size (bytes)  !! description
 |-
-| matchid    ||  int  || the match ID
+| orderid    || 16 || the order ID
+|-
+| matchid    || 8  || the ID assigned to this match
 |-
-| swap stage || string || one of "init" or "redeem"
+| tx hash    || 32  || the transaction hash
```

Comment:
I don't know enough about the spectrum of coins that can be supported with a UTXO model, but I'm concerned enforcing 32 byte tx hashes could be trouble.